### PR TITLE
Automate DEV setup for s3watcher

### PR DESCRIPTION
**Pull request by theefer**
Add utility scripts and helpers to make it easy to develop on the s3watcher locally, without having to fudge things manually.

See the updated README for details.

Note: There is still an issue with having to add a flag to allow insecure TLS connections locally (the loader cert is not validating properly).

### CODE REVIEW

File: s3watcher/lambda/package.json [hunk 1]
```diff
@@ -4,17 +4,20 @@
   "description": "",
   "main": "S3Watcher.js",
   "dependencies": {
+    "aws-sdk": "^2.2.21",
```
Comment (akash1810) [inline s3watcher/lambda/package.json]: should this be in dev deps? The aws sdk is already included in the lambda environment.

Comment (theefer) [inline s3watcher/lambda/package.json]: Ah OK that's why it was in dev deps. I'll remove it as it's now in the deps of the scripts package.json. Thanks!

Comment (theefer) [inline s3watcher/lambda/package.json]: Or maybe it's needed for local runs so will move back to dev deps..

**Comment by kenoir** [comment]
:+1: nice

**Comment by akash1810** [comment]
:+1:

**Pull request merged**